imports/plugins/core/i18n/client/components/localizationSettings.js

```diff
@@ -72,7 +72,7 @@ class LocalizationSettings extends Component {
   }
 
   renderCurrencies() {
-    return this.props.currencies.map((currency, key) => (
+    return this.props.currencies.sort((a, b) => a.name.localeCompare(b.name)).map((currency, key) => (
```

Comment:
This isn't a blocker at all, but I've been auditing `eslint warnings`, and short variable names is one of them. Can we update this to be `currencyA` and `currencyB`?